lib/mayor_game_web/templates/dashboard/show.html.heex

```diff
@@ -11,54 +11,65 @@
         <hr class="border-neutral-300 flex-1" />
         <span><%= rem(@world.day, 365) %></span>
       </div>
-      <div class="flex flex-row items-center space-x-1 max-w-sm">
-        <span>Cities</span>
+
+      <div class="flex flex-row items-center space-x-1 text-blue-800">
+        <hr class="border-neutral-300 flex-1" />
+        <span>World</span>
         <hr class="border-neutral-300 flex-1" />
-        <span>
-          <%= Number.Delimit.number_to_delimited(length(@cities)) %>
-        </span>
       </div>
-      <div class="flex flex-row items-center space-x-1 max-w-sm">
-        <span>Total pollution</span>
+      <div class="flex flex-row items-center space-x-1">
+        <div class="flex flex-col items-center gap-1 w-24">
+          <span>Cities</span>
+          <img src="/images/city.svg" alt="city" height="36" width="36" />
+          <span>
+            <%= Number.Delimit.number_to_delimited(length(@cities)) %>
+          </span>
+        </div>
+        <div class="flex-1" />
+        <div class="flex flex-col items-center gap-1 w-24">
+          <span>Population</span>
+          <img src="/images/citizen.svg" alt="city" height="36" width="36" />
+          <span>
+            <%= Number.Delimit.number_to_delimited(@citizen_sum) %>
```

Comment:
Where is @citizen_sum generated? I would expect this to be somewhere in `dashboard_live.ex`, but don't see any changes in that file in this PR

Reply:
They were in dashboard_live.ex but those changes were wiped in a merge commit 6530cb4. Technically this is not related to citizen fix (it belongs to lvm_ui_beautification) but somehow the three branches share the same code.